### ISSUE TITLE
Some input on documentation

### ISSUE THREAD

**Issue by brundles** [sardoj/VTModuleDesigner#79]
Hi again :)

Just a small note (or humble request ;)) on the documentation of this module (Which is important for people to use it + minimize stupid questions ;-))

I really like the youtube instructions!

But, it skips some of the menu-items, ie:
- Custom Links
- Related lists (this one I guessed)
- Events (Uh - seems exciting, but I cant figure out how to use them)
- Custom.

Also a MINI-intro to plugins etc. would be super.

Basically if you could perhaps just add a few bullets to each point, it would be enough to get people started. 

I also hate documenting myself, so its not that "we" want you to spend hours on it, but if you just invested additional 15 mins on describing the above in the readme or in the wiki, it would be a big help - and I can guess you already spend MANY hours on building this excellent module :-)

THANK you :) :)

**Comment by sardoj** [comment]
It's true, I spend months and months to make ModuleDesigner :-)
But you are right that it needs more help and documentation. You will be
able to find soon more tutorials in a specific website.

I will communicate soon about it.

**Comment by brundles** [comment]
:-)

Sounds good! Again - you dont have to spend tons of hours on it. Just spend 15 mins more in the wiki or read-me would probably be enough for 80% of the target group :)
